### PR TITLE
Rewrite TableRow component

### LINKED ISSUE #1101: Rewrite TableRow component
Remove `a` tag

### PR DESCRIPTION
**Pull request by sab94a**
## PR includes

- [x] Refactor

## Related issues

Resolve #1101

### CODE REVIEW

**Review by awinogradov — CHANGES_REQUESTED** [review]
https://styled-components.com/docs/api#as-polymorphic-prop

File: src/components/ProjectListItemCollapsable/ProjectListItemCollapsable.tsx [hunk 1]
```diff
@@ -127,7 +125,13 @@ export const ProjectListItemCollapsable: React.FC<ProjectListItemCollapsableProp
                                 <Text color={gray7}>{tr('No goals')}</Text>
                             )}
                         </StyledGoalsButtonContainer>
-                        {href && <StyledOpenButton size="s" onClick={onExternalLinkClick} />}
+                        {href && (
```
Comment (awinogradov) [inline src/components/ProjectListItemCollapsable/ProjectListItemCollapsable.tsx:128]: nullable

**Review by awinogradov — APPROVED** [review]

**Comment by sab94a** [comment]
issue #1224

**Comment by sab94a** [comment]
issue: #1135

**Pull request merged**